codec-http2/src/main/java/io/netty/handler/codec/http2/Http2FrameCodec.java

```diff
@@ -343,8 +346,8 @@ private void writeGoAwayFrame(ChannelHandlerContext ctx, Http2GoAwayFrame frame,
         int lastStreamCreated = connection().remote().lastStreamCreated();
         long lastStreamId = lastStreamCreated + ((long) frame.extraStreamIds()) * 2;
         // Check if the computation overflowed.
-        if (lastStreamId > Integer.MAX_VALUE) {
```

Comment:
It's sort of important that it was _Integer_.MAX_VALUE, as Long.MAX_VALUE wouldn't work. I'd prefer this stay explicit.

Reply:
sgtm ... reverted.